<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DungeonGeneratorClass</class>
 <widget class="QMainWindow" name="DungeonGeneratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dungeon Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DungeonGenerator.qrc">
    <normaloff>:/DungeonGenerator/Resources/maze.png</normaloff>:/DungeonGenerator/Resources/maze.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_debug">
        <property name="text">
         <string>Debug</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_zoomOut">
        <property name="toolTip">
         <string>Zoom Out</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="DungeonGenerator.qrc">
          <normaloff>:/DungeonGenerator/Resources/magnifier_zoom_out.png</normaloff>:/DungeonGenerator/Resources/magnifier_zoom_out.png</iconset>
        </property>
        <property name="shortcut">
         <string>-</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_zoomIn">
        <property name="toolTip">
         <string>Zoom In</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="DungeonGenerator.qrc">
          <normaloff>:/DungeonGenerator/Resources/magnifier_zoom_in.png</normaloff>:/DungeonGenerator/Resources/magnifier_zoom_in.png</iconset>
        </property>
        <property name="shortcut">
         <string>+</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_New">
        <property name="toolTip">
         <string>Generate Maze</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="DungeonGenerator.qrc">
          <normaloff>:/DungeonGenerator/Resources/maze.png</normaloff>:/DungeonGenerator/Resources/maze.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Dungeon">
        <property name="toolTip">
         <string>Generate Dungeon!</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="DungeonGenerator.qrc">
          <normaloff>:/DungeonGenerator/Resources/dungeon.png</normaloff>:/DungeonGenerator/Resources/dungeon.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DungeonGenerator.qrc"/>
 </resources>
 <connections/>
</ui>
